Xamarin.Forms.Platform.Android/Renderers/ScrollViewRenderer.cs

```diff
@@ -373,6 +373,10 @@ async void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 			while (IsLayoutRequested)
 			{
 				await Task.Delay(TimeSpan.FromMilliseconds(1));
+				
+				if (_disposed)
```

Comment:
I think it's better to check out native `IsDisposed` here. Since NRE can occur on call `IsLayoutRequested` method.